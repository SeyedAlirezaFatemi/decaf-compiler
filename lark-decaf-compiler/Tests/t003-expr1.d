int main() {
    int a;
    int b;
    int c;
    int d;

    int z;
    
    a = ReadInteger();
    b = ReadInteger();
    c = ReadInteger();
    d = ReadInteger();

    z = a + (b * 5);
    a = z * d;
    z = 2 * a + ((a + b) / (c + d));
    b = z / a;
    c = b * a + z;
    d = a - b - c - d - z;

    Print(a);
    Print(b);
    Print(c);
    Print(d);

    Print(z);
}