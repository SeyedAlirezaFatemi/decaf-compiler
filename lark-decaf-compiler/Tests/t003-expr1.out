105
2
428
-550
218
